extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/ReactivePersistenceUnit.java

```diff
@@ -0,0 +1,25 @@
+package io.quarkus.hibernate.orm;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PACKAGE;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import jakarta.inject.Qualifier;
+import jakarta.persistence.EntityManagerFactory;
+
+/**
+ * This is used to qualify the {@link EntityManagerFactory} that should be used for Hibernate Reactive.
+ */
+@Target({ TYPE, FIELD, METHOD, PARAMETER, PACKAGE })
+@Retention(RUNTIME)
+@Documented
+@Qualifier
+public @interface ReactivePersistenceUnit {
```

Comment:
Putting this here so we don't forget: 


> > > There's also a new ReactivePersistenceUnit annotation https://github.com/quarkusio/quarkus/compare/main...lucamolteni:13425-ter?expand=1#diff-8c10ee8583155144d5c9f796e33b785471701a346c768b2f707fe3a036f0590dR24 in order to disambiguate the correct bean to be injected.
> > 
> > 
> > Can this be more "hidden"? E.g. move the annotation to a clearly internal package, or even just don't make it a bean for Hibernate Reactive.
> > The reason I'm asking is that users really shouldn't be using the `EntityManagerFactory` for Hibernate Reactive, and thus `@ReactivePersistenceUnit` really has no use except internally, which could be confusing.
> 
> I agree with your point, I'll work on it and see if I can hide it somehow. But yes it's just a matter of CDI injection (answering also 1 below)

Reply:
I've removed the annotation with https://github.com/quarkusio/quarkus/pull/44473/commits/fc74c77d111e8f2d627716d60b29d0fb9e2f5a22 take a look please